tests/e2e/cluster_test.go

```diff
@@ -338,6 +339,9 @@ func (cfg *etcdProcessClusterConfig) etcdServerProcessConfigs(tb testing.TB) []*
 		if cfg.CompactHashCheckTime != 0 {
 			args = append(args, "--experimental-compact-hash-check-time", cfg.CompactHashCheckTime.String())
 		}
+		if cfg.WatchProcessNotifyInterval != 0 {
+			args = append(args, "--experimental-watch-progress-notify-interval", cfg.WatchProcessNotifyInterval.String())
+		}
```

Comment:
Minor comment: this seems not in the original PR.

Reply:
Yes, added a commit for backport that adds missing feature in e2e testing framework